### ISSUE TITLE
Searchable opds list - new feature - is overriden by custom list in settings

### ISSUE THREAD

**Issue by ptrm** [koreader/koreader#5830]
* KOReader version: v2020.01-20-g67627_2020-02-08
* Device: Kobo Forma

#### Issue
The list from opds widget's `init()` gets overriden, not merged, by the one from settings.reader.lua::["opds_servers”]. That's probably desirable, but after introducing #5823 might confuse users. Confused me at least, until I grepped for the other one and removed it ;)

Maybe it would be a good idea to save only new elements in settings, and merge them, ignoring duplicates, with the hardcoded one? Instead of replacing the whole list as it is now.

#### Steps to reproduce
Having a pre- #5823 version, add a custom opds server via gui.

Update to the recent dev build. The list will be the one present before update.

**Comment by Frenzie** [comment]
> Maybe it would be a good idea to save only new elements in settings, and merge them, ignoring duplicates, with the hardcoded one? Instead of replacing the whole list as it is now.

Not quite, people presumably want to be able to remove what they consider default random stuff. 

Probably introduced in https://github.com/koreader/koreader/pull/2508

**Comment by ptrm** [comment]
> Not quite, people presumably want to be able to remove what they consider default random stuff.

True. Anyway, I think something to draw user's attention to e.g. new searchable servers or any other new ones might be useful.
